### PR TITLE
[Merged by Bors] - chore(Order/Interval/Finset/Defs): golf

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/Interval/Finset/Defs.lean [hunk 1]
```diff
@@ -712,58 +712,42 @@ instance OrderDual.instLocallyFiniteOrder : LocallyFiniteOrder αᵒᵈ where
 
 theorem Icc_toDual : Icc (toDual a) (toDual b) = (Icc b a).map toDual.toEmbedding := by
   refine' Eq.trans _ map_refl.symm
-  ext c
-  rw [mem_Icc, mem_Icc (α := α)]
-  exact and_comm
+  rfl
```
Comment (YaelDillies) [inline Mathlib/Order/Interval/Finset/Defs.lean:715]: Maybe
```
theorem Icc_toDual : Icc (toDual a) (toDual b) = (Icc b a).map toDual.toEmbedding := map_refl.symm
```
even?

Comment (mo271) [reply Mathlib/Order/Interval/Finset/Defs.lean]: That's even better

**Review by YaelDillies — APPROVED** [review]
maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors d+

**Pull request closed without merging**